example-crates/wallet_esplora_async/src/main.rs

```diff
@@ -1,25 +1,47 @@
 use std::{collections::BTreeSet, io::Write, str::FromStr};
+use tokio::runtime::Handle;
 
 use bdk_esplora::{esplora_client, EsploraAsyncExt};
 use bdk_wallet::{
     bitcoin::{Address, Amount, Network, Script},
     KeychainKind, SignOptions, Wallet,
 };
 
-use bdk_sqlite::{rusqlite::Connection, Store};
+// use bdk_sqlite::{rusqlite::Connection, Store};
+use bdk_postgres;
+use bdk_postgres::Store;
+use bdk_wallet::bitcoin::secp256k1::Secp256k1;
+use bdk_wallet::wallet::wallet_name_from_descriptor;
+use postgres::{Client, Config, NoTls};
```

Comment:
I don't mind having a Postgres example, but we shouldn't omit/avoid SQLite examples.

Reply:
we wont, the sqlite example would get moved to a non async example since this crate uses the tokio runtime